app/scripts/modules/core/src/application/ApplicationPostSearchResultSearcher.ts

```diff
@@ -0,0 +1,45 @@
+import { IPromise } from 'angular';
+
+import { ApplicationReader, IApplicationSummary } from './service/application.read.service';
+import { IPostSearchResultSearcher } from 'core/search/searchResult/PostSearchResultSearcherRegistry';
+import { ISearchResult } from 'core/search/search.service';
+import { ISearchResultFormatter, searchResultFormatterRegistry } from 'core/search/searchResult/searchResultFormatter.registry';
+import { ISearchResultSet } from 'core/search/infrastructure/infrastructureSearch.service';
+import { IServerGroupSearchResult } from 'core/search/searchResult/model/IServerGroupSearchResult';
+
+export class ApplicationPostSearchResultSearcher implements IPostSearchResultSearcher<IServerGroupSearchResult> {
+
+  private static TYPE = 'applications';
+
+  constructor(private applicationReader: ApplicationReader) {}
```

Comment:
I think you need `'ngInject';` in here

Reply:
The class is being constructed and registered in a `.run()` block rather than actually injected using angular DI so I _think_ this is right the way it is.  Am I missing something?